package.json

```diff
@@ -128,6 +128,7 @@
         "redux": "^4.0.1",
         "redux-perf-middleware": "^1.2.2",
         "regenerator-runtime": "^0.13.2",
+        "resolve-url-loader": "^3.1.2",
```

Comment:
Where are you using this in the website?

Reply:
we were using it in our webpack config but we're actually not anymore